java/test/jmri/jmrix/cmri/serial/cmrinetmetrics/CMRInetMetricsCollectorTest.java

```diff
@@ -0,0 +1,44 @@
+package jmri.jmrix.cmri.serial.cmrinetmetrics;
+
+import apps.tests.Log4JFixture;
+import jmri.util.JUnitUtil;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+import java.awt.GraphicsEnvironment;
+
+/**
+ * Test simple functioning of CMRInetMetricsCollector
+ *
+ * @author	Chuck Catania Copyright (C) 2017
+ */
+public class CMRInetMetricsCollectorTest {
+
+    @Test
+    public void testStringCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        CMRInetMetricsCollectorTest action = new CMRInetMetricsCollectorTest(); 
+        Assert.assertNotNull("exists", action);
+    }
+
+    @Test
+    public void testCtor() {
+        Assume.assumeFalse(GraphicsEnvironment.isHeadless());
+        CMRInetMetricsCollectorTest action = new CMRInetMetricsCollectorTest(); 
```

Comment:
Did you intend for this to create a new *Test object? Usually this test is creating the object under test itself, i..e CMRInetMetricsCollector.

Reply:
What is missing are tests  CMRInetMetricsAction and CMRInetMetricsFrame under cmirnetmanager. I will create those.